package.json

```diff
@@ -9,8 +9,8 @@
     "concurrently": "^8.2.2",
     "copy-production-deps": "^0.2.1",
     "cross-env": "^7.0.3",
-    "nw": "^0.79.0-sdk",
-    "nw-builder": "latest",
+    "nw": "0.79.0-sdk",
+    "nw-builder": "^4.4.2-beta.3",
```

Comment:
Because the build code is based on this version of NW-Builder, and its API may change in the future, it is better to show boilerplate uses the version you were using, so they have a known-good.